frontend/src/metabase-lib/v1/expressions/pratt/common.ts

```diff
@@ -1,89 +0,0 @@
-import type { Expression } from "metabase-types/api";
```

Comment:
These test utils are no longer needed, and I removed them because they were confusingly defined in a non-test file.